mbuild/utils/io.py

```diff
@@ -99,7 +99,26 @@ def import_(module):
     >>> # module from, etc) if the import fails
     >>> import tables
     >>> tables = import_('tables')
+
+    Notes
+    -----
+    The pybel/openbabel block is meant to resolve compatibility between
+    openbabel 2.x and 3.0.  There may be other breaking changes but the change
+    in importing them is the major one we are aware of. For details, see
+    https://open-babel.readthedocs.io/en/latest/UseTheLibrary/migration.html#python-module
     """
+    if module == 'pybel':
+        try:
+            return importlib.import_module('openbabel.pybel')
+        except ModuleNotFoundError:
+            pass
+        try:
+            pybel = importlib.import_module('pybel')
+            raise DeprecationWarning('openbabel 2.0 detected and will be '
+                    'dropped in a future release. Consider upgrading to 3.x.')
```

Comment:
Do you need to do something like `warnings.warn(msg)`? The unit tests error out and exit when `DeprecationWarning` is raised.